packages/rocketchat-lib/server/models/_BaseCache.js

```diff
@@ -744,6 +747,154 @@ class ModelsBaseCache extends EventEmitter {
 		}
 	}
 
+	updateRecord(record, update) {
```

Comment:
should we make clear we're compatible with a specific mongodb version? since we have now support to specific operators (e.g.: `$set`, `$unset`, `$min`, etc).

if mongodb adds a new operator, we'll have to add specific support to it, even if Meteor already supports.